gear-test/src/address.rs

```diff
@@ -17,10 +17,31 @@
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
 use gear_core::program::ProgramId;
+use once_cell::sync::Lazy;
+use primitive_types::H256;
 use serde::{Deserialize, Deserializer, Serialize};
-use sp_core::{crypto::Ss58Codec, hexdisplay::AsBytesRef, sr25519::Public, H256};
-use sp_keyring::sr25519::Keyring;
-use std::str::FromStr;
+use std::collections::HashMap;
+
+static ACCOUNTS: Lazy<HashMap<&'static str, H256>> = Lazy::new(|| {
+    fn public_key(s: &'static str) -> H256 {
+        H256::from_slice(hex::decode(s).unwrap().as_slice())
+    }
+
+    let mut accounts = HashMap::new();
+    accounts.insert(
+        "alice",
+        public_key("d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d"),
+    );
+    accounts.insert(
+        "bob",
+        public_key("8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48"),
+    );
+    accounts.insert(
+        "eve",
+        public_key("e659a7a1628cdd93febc04a4e0646ea20e9f5f0ce097d9a05290d4a9e054df4e"),
+    );
+    accounts
```

Comment:
Isn't it better to convert an array of tuples to `HashMap`?

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=0064b67bf14fa9153a367fcf6f963e51

Reply:
You are right. It will look cleaner